### DISCUSSION TITLE
An error occurred running the project

### DISCUSSION THREAD

**Discussion by ebazhanglang** [babel/babel#15378] [Q&A]
hi!
  The following is the screenshot of the error, please help to have a look
![image](https://user-images.githubusercontent.com/110316089/215427223-2731cb7d-6919-40a7-86be-95ebee903e57.png)

**Comment by liuxingbaoyu** [comment]
You are using a version of babel that is many years old, please update to the latest version.

**Comment by ebazhanglang** [comment]
But&nbsp;won't&nbsp;there&nbsp;be&nbsp;any&nbsp;compatibility&nbsp;problems&nbsp;with&nbsp;suddenly&nbsp;upgrading&nbsp;to&nbsp;the&nbsp;latest&nbsp;version?


---Original---
From: ***@***.***&gt;
Date: Mon, Jan 30, 2023 21:08 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [babel/babel] An error occurred running the project (Discussion#15378)




 
You are using a version of babel that is many years old, please update to the latest version.
 
&mdash;

**Comment by limingcan562** [comment]
`corejs`这个配置项，在`Babel 7.4.0`以后才被加入，目前你的`Babel`版本太老了，得升级一下

我感觉你对`Babel`可能没有一个系统的了解，可以看看我这篇文章 [想弄懂Babel？你必须得先弄清楚这几个包](https://juejin.cn/post/7190312484492804156)
